test/template.rb

```diff
@@ -3,7 +3,6 @@
 copy_file "test/support/circleci.rb"
 copy_file "test/support/mailer.rb"
 copy_file "test/support/rails.rb"
-copy_file "test/support/sidekiq.rb"
```

Comment:
ActiveJob has a robust set of test helpers. We can test our jobs via those helpers, instead of relying on Sidekiq directly.